admin-styles.php

```diff
@@ -0,0 +1,234 @@
+<style type="text/css">
```

Comment:
Why not make this a `css` file and load via [`wp_enqueue_style`](https://developer.wordpress.org/reference/functions/wp_enqueue_style/)?